### ISSUE TITLE
Time range in bing engine not working

### ISSUE THREAD

**Issue by Mena489** [searxng/searxng#3358]
**Version of SearXNG, commit number if you are using on master branch and stipulate if you forked SearXNG**
b21aaa89075b325bd89d2f9f5dc82bbb66855d12

**How did you install SearXNG?**
just built the docker image using Dockerfile

**What happened?**
When choosing time range like last year and last month it gives results outside of the range

**Comment by Mena489** [comment]
It's likely related to this bug
https://github.com/searxng/searxng/issues/2641

**Comment by Mena489** [comment]
Edit : It works but gets irrelevant results and when try the same bing api url on my browser it gets accurate results

**Comment by Mena489** [comment]
Edit 2 : it happens when using double qoutes in the query

**Comment by return42** [comment]
Yeah, bing is a source of constant errors / I tagged some of them: https://github.com/searxng/searxng/labels/bing-engine

Personally I gave up to fix this engine, we will probably never find a stable solution for this, because bing itself is already full of quirks https://github.com/searxng/searxng/issues/2641#issuecomment-1742526819 (Seems bing does not work in a VPN).

**Comment by return42** [comment]
strong related:

- https://github.com/searxng/searxng/issues/3402

- [information on the parameters](https://github.com/searxng/searxng/pull/3416#issuecomment-2344841069)
